Engine/Entities/Entity.cs

```diff
@@ -29,23 +32,29 @@ public Entity(string name, Transform transform, World world)
         /// Names are not necessarily unique but it's better to distinguish entities. 
         /// </summary>
         public string Name { get; private set; }
-        
+
         /// <summary>
         /// Transform of the entity in the world.
         /// Represents the position, rotation and scale of the entity in the world.
         /// </summary>
         public Transform Transform { get; private set; }
-        
+
         /// <summary>
         /// World of the entity.
         /// </summary>
         public World World { get; private set; }
 
+        public (int X, int Y) Region
+        {
+            get => _region;
+            set => _region = value;
+        }
```

Comment:
couldn't this be an auto property ?

Reply:
It could, I decided to use the more verbose form which doesn't change anything performance-wise